src/components/NewMessage/NewMessage.vue

```diff
@@ -54,6 +54,10 @@
 
 			<!-- Input area -->
 			<div class="new-message-form__input">
+				<NewMessageAbsenceInfo v-if="userAbsence"
```

Comment:
I wasn't sure if it should be from the server side to only render the active OoO or any OoO that exists because dates were given in the response. 

Should we use them to show when they will be back for example ? like on Webex

Reply:
In the next iteration maybe